### PR TITLE
Add example of how to customise Jenkinsfile

### PR DESCRIPTION
**Pull request by PhilipGough**
ping @laurafitzgerald

### CODE REVIEW

File: docs/digger/developer/jenkinsfile-create.asciidoc [hunk 1]
```diff
@@ -306,3 +306,47 @@ against. If empty, the SDK will be determined by XCode.
...
+      ...
+  }
+  ...
+}
+----
+
+
+==== Installing and using a specific dependency version
+
+A developers Cordova application can only be built with Cordova version
```
Comment (laurafitzgerald) [inline docs/digger/developer/jenkinsfile-create.asciidoc]: developers -> developer's

Comment (laurafitzgerald) [inline docs/digger/developer/jenkinsfile-create.asciidoc]: Could we expand on the description here?

**Comment by PhilipGough** [comment]
@laurafitzgerald comments fixed. Can be tested with anything that comes to mind but nothing concrete here. Just an example. Feel free to try and break it. Node version needs the new slave image alright. npm dependency should work as is

File: docs/digger/developer/jenkinsfile-create.asciidoc [hunk 2]
```diff
@@ -306,3 +306,47 @@ against. If empty, the SDK will be determined by XCode.
 | Path to the app to sign.
 |===
 
+
+=== Customising Jenkinsfile
+
+A developer may wish to customise their `Jenkinsfile` to build an application. For instance they may to install
```
Comment (laurafitzgerald) [inline docs/digger/developer/jenkinsfile-create.asciidoc]: may to -> may want to

**Comment by laurafitzgerald** [comment]
@PhilipGough Could we consider adding this to the sample Jenkinsfiles as well. i.e. use a variable defined at the beginning of the file. If it's provided run the command to install the required dependency?
e.g. if CORDOVA_VERSION then sh "npm install cordova@$CORDOVA_VERSION"

**Comment by PhilipGough** [comment]
@laurafitzgerald do you mean as an example? I'm not sure how useful it will be since the cordova version is just an example. They should be able to apply this to anything. For example if they want to do some build with `grunt` etc they could use the same approach. 

I don't think it is viable to cover every use case in the Jenkinsfile and again its just to show users how the might customise a build.

**Comment by laurafitzgerald** [comment]
@PhilipGough cool makes sense and update look good.

**Pull request merged**